the-blue-alliance-ios/View Controllers/Districts/DistrictsContainerViewController.swift

```diff
@@ -1,14 +1,14 @@
 import Foundation
 import UIKit
 import TBAKit
+import FirebaseRemoteConfig
 
 private let SelectYearSegue = "SelectYearSegue"
 
 class DistrictsContainerViewController: ContainerViewController {
 
-    // TODO: Get these from Firebase Config
-    var maxYear: Int = 2018
-    var year: Int = 2018 {
+    var maxYear: Int = RemoteConfig.remoteConfig().maxSeason
```

Comment:
Eventually this singleton access will go away, but that won't be until we start doing constructor injection, which we need constructors for, which we need to remove storyboards for